Add Boost Development page

GregKaleka opened the pull request:
Fixes #1559 

![Screenshot 2024-12-20 at 10 21 05 AM](https://github.com/user-attachments/assets/04201ff0-3568-4e2d-b0cb-77c7b4613910)
![Screenshot 2024-12-20 at 10 24 57 AM](https://github.com/user-attachments/assets/1fc181f2-ad07-4965-a621-5608f95b5207)

rbbeeston commented on templates/boost_development.html:
```diff
@@ -0,0 +1,31 @@
+<div class="flex justify-center items-center my-4">
+    <h1 class="text-2xl">Development</h1>
+</div>
+<div class="container px-4 mb-8 mx-auto">
```
can we add the 'card' class here to wrap the top part in a card, and have less of the grey background?

rbbeeston commented:
A couple of minor changes:

1. let's match the height of the header font to 3xl and align it left
2. Wrap the content using the 'card' class as shown in the image.

![image](https://github.com/user-attachments/assets/e9cbd6e1-b30d-4d8e-bbaa-374a9c11bfe6)

GregKaleka commented:
Better?

<img width="1134" alt="Screenshot 2024-12-23 at 2 47 49 PM" src="https://github.com/user-attachments/assets/b2da2a4d-ab1a-45b1-adf6-fabeb3fe71d0" />

GregKaleka commented:
Fixed width issue, and also noticed the widgets needed to wrap on mobile, so I added some responsive classes.

![Screenshot 2024-12-23 at 3 26 16 PM](https://github.com/user-attachments/assets/552aeaeb-50f9-4f47-83bd-ea804dacf2df)
![Screenshot 2024-12-23 at 3 26 30 PM](https://github.com/user-attachments/assets/249cadc9-43cb-4c0e-9c64-c1bfcfb44572)

rbbeeston reviewed (approved):
LGTM!
